webpage.html

```diff
@@ -0,0 +1,282 @@
+                    <h3 id="supernova">Supernova</h3>
+                    <iframe width="560" height="315" src="https://www.youtube.com/embed/phuiiNCxRMg?si=72FXyHxuAOUmQJwV" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+                    <blockquote>
+                        "The 'futuristic' music video shows Aespa
+                        in 'dynamic and witty characterizations',
+                        displaying their superpowers throughout
+                        the city. It features each member possessing
+                        an 'otherworldly power' - Karina's super
+                        strength, Ningning's control over fire,
+                        Winter's floating in the air and Giselle's
+                        time-looping. The visual starts with Karina
+                        surviving a fall from way high up onto a car.
+                        Moments later, she leaves the shattered vehicle
+                        unharmed... Additionally, AI
+                        technology was used to move still photos
+                        of Aespa's faces in synergy with the lyrics
+                        of the song."
+                        <footer>
+                            - Source from <cite><a href="https://en.wikipedia.org/wiki/Supernova_(Aespa_song)">Wikipedia</a></cite>
+                        </footer>
+                    </blockquote>
+                </section>
+            </article>
+            <aside>
+                <form>
+                    <label for="aespa-songs-vote">Vote: Which is your favorite AESPA song?</label>
+                    <select name="aespa-songs" id="aespa-songs-vote">
+                        <option value="black-mamba-option">Black Mamba</option>
+                        <option value="next-level-option">Next Level</option>
+                        <option value="girls-option">Girls</option>
+                        <option value="my-world-option">Welcome to MY World</option>
+                        <option value="drama-option">Drama</option>
+                        <option value="supernova-option">Supernova</option>
+                    </select>
+                    <button type="submit">Submit</button>
+                </form>
+            </aside>
+            <article>
+                <h2 id="achievements">AESPA's Achievements</h2>
+                <h3>Chart History:</h3>
+                <figure>
+                    <img src="https://pbs.twimg.com/media/GHPjPI2XYAASZa1.png" alt="AESPA's Next Level Ranks #11 on the longest charting songs by a KPOP group.">
+                    <figcaption>Longest Charting Songs by a KPOP Group (Updated 2024)</figcaption>
+                </figure>
+                <h3>Music Show Wins:</h3>
+                <table>
+                    <thead>
```

Comment:
We need a `<caption>` below `<table>` so that the screenreader has a caption for the table. It is ok to have the same text that you use in the `<h3></h3>`. I would recommend to add  `<caption>Music Show Wins</caption>` below `<table>` and before `<thead>`